spyder/utils/pixi.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Conda/anaconda utilities."""
+
+# Standard library imports
+from functools import lru_cache
+import os
+
+# Third-party imports
+from packaging.version import parse
+
+# Local imports
+from spyder.utils.programs import find_program, run_program
+
+WINDOWS = os.name == "nt"
+
+
+def find_pixi(pyexec=None):
+    """
+    Find pixi executable.
+
+    `pyexec` is a python executable, the relative location from which to
+    attempt to locate a pixi executable.
+    """
+    # Try the environment variables
+    pixi_home = os.environ.get('PIXI_HOME')
+    pixi = None
+    if pixi_home:
+        pixi = os.environ.get('PIXI_HOME') or os.environ.get('MAMBA_EXE')
```

Comment:
```suggestion
        pixi = os.environ.get('PIXI_HOME')
```

I don't think we should use mamba to activate pixi envs because the logic is different.